schutzbot/ci_details.sh

```diff
@@ -37,13 +37,6 @@ echo "List of installed packages:"
 rpm -qa | sort
 echo "------------------------------------------------------------------------------"
 
-if rpm --quiet -q python36; then
-    echo -e "\n FAIL: python36 is installed, see #794 ..."
-    exit 1
-else
-    echo -e "\n PASS: python36 not insalled"
-fi
```

Comment:
This check was targeting a specific issue and we don't want to have python36 (not sure if that applies to other versions) pre-installed into CI job runners.  This should be removed unless we know better.

Reply:
Uhh I'm not sure I understand now. I removed it because it sounded obsolete, should I keep it that way or bring it back?

Reply:
Bring it back, b/c at this point in time we don't know if this snippet needs to be removed or is still necessary.